components/bluetooth/lib.rs

```diff
@@ -735,17 +754,21 @@ impl BluetoothManager {
                                 self.allowed_services
                                     .get(&device_id)
                                     .map_or(false, |uuids| uuids.contains(&s.uuid)));
+
+        // Step 2.7: If its argument is empty, throws a NotFoundError.
         if services_vec.is_empty() {
             return drop(sender.send(Err(BluetoothError::NotFound)));
         }
 
         return drop(sender.send(Ok(BluetoothResponse::GetPrimaryServices(services_vec))));
     }
 
+    // https://webbluetoothcg.github.io/web-bluetooth/#dom-bluetoothremotegattservice-getincludedservice
```

Comment:
We really need to link to https://webbluetoothcg.github.io/web-bluetooth/#getgattchildren somewhere too.